tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,448 @@
+
+- Cloud-based platform for planetary scale geospatial analysis
+- Uses Google's computational resources to reduce processing time
+- Massive archive of remote sensing data 
+- 200+ public datasets 
+- greater than 4000 new images every day 
+- greater than 5 million images 
+- greater than 5 petabytes of data
+
+Source: Google Earth Engine User summit
+
+---
+
+![Code Editor (Source: developers.google.com)](ee-editor.png)
+
+---
+
+## Basic Functions
+
+### Declaring variables
+```javascript
+var varName = Containerforvariabletype(variable name); 
+```
+A container (in the form ee.variabletype) is used to wrap up a native javascript object type so that Google's server can recognize its structure and perform operations on it
+
+###  Centering map
+```javascript
+Map.setCenter(long, lat, zoom level);
+```
+>Zoom level varies from 0 (no zoom) to 20 (highest zoom level)
+
+### Displaying metadata
+```javascript
+print(varName)
+```
+You cannot print more than 5000 elements at once
+
+### Adding a layer to the map
+```javascript
+Map.addLayer(varName);
+```
+
+---
+
+## Variable types in Earth Engine
+
+### Strings
+
+```javascript
+var str = ee.String("This is a string. Or is it? It is.");
+```
+### Numbers
+```javascript
+var num = ee.Number(5);
+```
+
+### Arrays
+```javascript
+var arr = ee.Array([[5, 2, 3],  [-2, 7, 10],  [6, 6, 9]]);
+```
+
+### Lists
+```javascript
+var lis = ee.List([5, "five" , 6, "six"]);
+```
+### Dictionaries
+```javascript
+var dict = ee.Dictionary({five: 5 , six: 6});
+```
+
+### And the fun stuff
+- Geometries
+- Features
+- Feature Collections
+- Images
+- Image Collections
+
+---
+
+## Geometries – declaration and types
+
+### Points
+```javascript
+var poi = ee.Geometry.Point(0, 45);
+```
+### Multi Points
+```javascript
+var multi = ee.Geometry.MultiPoint(0, 45, 5,6, 70,-56);
+```
+### Line String
+```javascript
+var linestr = ee.Geometry.LineString([[0, 45], [5,6], [70,-56]]);
+```
+### Multi Line String
+```javascript
+var mlinestr = ee.Geometry.MultiLineString([[[0, 45], [5,6], [70,-56]], [[0, -45], [-5,-6], [-70,56]]]);
+```
+### Linear Ring
+```javascript
+var linrin = ee.Geometry.LinearRing(0, 45, 5,6, 70,-56, 0,45);
+```
+### Rectangle
+```javascript
+var rect = ee.Geometry.Rectangle(0, 0, 60,30);
+```
+### Polygon
+```javascript
+var poly = ee.Geometry.Polygon([[[0, 0], [6,3], [5, 5], [-30,2], [0,0]]]);
+```
+### Multi Polygon
+```javascript
+var multiploy = ee.Geometry.MultiPolygon([ee.Geometry.Polygon([[0, 0], [6, 3], [5, 5], [-30, 2], [0,0]]), ee.Geometry.Polygon([[0, 0], [-6, -3], [-5, -5], [30, -2], [0, 0]])]);
+```
+
+---
+
+## Features and Feature Collections
+
+- Features are geometries associated with specific properties
+- Feature Collections are groups of features
+
+
+![Chicago map by neighborhood](chicago-census.png)
+
+---
+
+## Functions and mapping
+
+- A set of instructions to perform a specific task
+```javascript
+function  function_Name(Arguments) {statements};
```

Comment:
Single space after "function"

no underscores is JavaScript symbol names

Reply:
Fixed.